src/shared/css/theme.css

```diff
@@ -2,14 +2,24 @@
 
 :root {
   /* App colors. */
-  --primary-color: #546e7a;
+  /* --primary-color: #546e7a; */
+  --primary-color: #ffffff;
   --primary-color-dark: #29434e;
   --primary-color-light: #819ca9;
   --secondary-color: #e0e0e0;
   --secondary-color-dark: #aeaeae;
   --secondary-color-light: #ffffff;
+  --secondary-color-jan:  #1a73e8;
```

Comment:
As discussed, don't create variables that have a specific meaning.What is this color for? Rename or delete what it's replacing